src/classes/i18n.js

```diff
@@ -162,7 +160,7 @@ export default class i18n {
 
     load_files (cb) {
 
-        this.translations = importer.dirloadSync(this.locales_folder);
+        this.translations = importer.dirloadSync(this.locale_folder);
         this.debug('from files', Object.keys(this.translations));
```

Comment:
I wonder if we should throw/log error if there's no translation ready?

Reply:
there is a checking on this already (see function is_valid_translation)